searx/settings.yml

```diff
@@ -2367,6 +2367,16 @@ engines:
     shortcut: pgo
     disabled: true
 
+# NixOS Package Search
+# - name: nixpkgs
+#   shortcut: nix
+#   engine: nixpkgs
+#   base_url: https://search.nixos.org/backend
+#   username:
+#   password:
+#   index: latest-42-nixos-unstable
```

Comment:
I am not aware of any other deployment out in the wild, so we could just omit those 4 options.